pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = [
+    "scikit-build-core",
+    "swig",
+    "numpy",
+    "pint",
+    "pyqt5",
+    "pyqtgraph",
+    "scipy",
+]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "pairinteraction"
+version = "0.9.9"
+description = "A Rydberg Interaction Calculator"
+license = { text = "GPL-3.0-or-later OR LGPL-3.0-or-later" }
+authors = [
+    { name = "Sebastian Weber" },
+    { name = "Henri Menke" },
+]
+homepage = "https://www.pairinteraction.org"
+repository = "https://github.com/pairinteraction/pairinteraction"
+requires = [
+    "numpy",
+    "pint",
+    "pyqt5",
+    "pyqtgraph",
+    "scipy",
+]
+
+[tool.scikit-build]
+wheel.packages = []
+cmake.verbose = true
+
+[tool.cibuildwheel]
+build = "cp38-manylinux_x86_64"
+build-verbosity = 1
+manylinux-x86_64-image = "ghcr.io/pairinteraction/pairinteraction-manylinux:docker"
```

Comment:
Maybe figure out how to vendor most libraries, then we can use any Docker image (use [Conan](https://conan.io/)?).

Reply:
Also Windows and macOS wheels. Needs restructuring of all the branches for installation.